### DISCUSSION TITLE
transfer learned spam

### DISCUSSION THREAD

**Discussion by ThorstenUm** [rspamd/rspamd#5196] [Ideas]
Hello,

is it actually possible to transfer learned spam and ham from one system to another system?
The background is that the system has to learn everything again when you set up a new mail server.
I haven't been able to figure out how to transfer the databases from one RSpamd to another RSpamd.

Greetings and thanks

**Comment by fatalbanana** [comment]
Yes, the key thing is to copy Redis data.

https://rspamd.com/doc/faq.html#what-data-should-be-backed-up

**Reply by ThorstenUm** [reply]
Thank you for the help.
How to perform recovery on the target system if databases already exist? Can the previously contained data simply be deleted after stopping the service?

Is it enough to just swap the Redis database or do the RSpamd databases also have to be swapped?
